packages/quantic/force-app/main/default/lwc/quanticSearchBox/templates/expandableSearchBox.css

```diff
@@ -27,7 +27,6 @@ textarea.searchbox__input {
 }
 
 .searchbox_floating-container {
-  position: absolute;
```

Comment:
I'm pretty sure I needed this for a good reason though :D 

I'm thinking it might be to work correctly in the header of the SF community as explained in the other comment.

To make it so the expansion of the searchbox doesn't expand also the header.

Reply:
yeah that would make a lot of sense

Reply:
<img width="1087" alt="image" src="https://github.com/coveo/ui-kit/assets/73316533/e278909d-b1a0-4302-bc3b-a1e938cf9baa">
it seems to work